tests/encoder_test.py

```diff
@@ -1,10 +1,15 @@
-import os 
+import sys, os 
 import unittest
 import numpy as np
-from neurogen import encoder 
+import struct
+import shutil
+from neurogen import encoder
+sys.path.insert(1, '../src/neurogen/')
```

Comment:
you should not need this sys.path. It should work with a standard import like I've done with encoder.